### ISSUE TITLE
[Tab] marking tabs 'active' when using dropdown menu

### ISSUE THREAD

**Issue by king2** [Semantic-Org/Semantic-UI#5664]
I'm using tabs in main menu to get main html loaded only once, and load main content from server on demand. So, I added code for menu:
```
<div class="ui dropdown item">
    <i class="settings icon"></i>Settings<i class="dropdown icon"></i>
    <div class="menu">
        <a class="tabitem item" data-tab="branch"><i class="building icon"></i>Branch</a>
        <a class="tabitem item" data-tab="shops"><i class="world icon"></i>Shops</a>
    </div>
</div>
<a class="tabitem item" data-tab="clients"><i class="users object icon"></i>Clients</a>
<a class="tabitem item" data-tab="orders"><i class="users object icon"></i>Orders</a>
```
and code for content:
```
<div id="main_content" class="ui container">
   <div class="ui tab" data-tab="branch"></div>
   <div class="ui tab" data-tab="shops"></div>
   <div class="ui tab" data-tab="clients"></div>
   <div class="ui tab" data-tab="orders"></div>
</div>
```
Init:
```
$('.admin_menu .tabitem').tab({
    history: true,
    historyType: 'hash',
    context: '#main_content',
    auto: true,
    debug: true,
    path: '/admin/'
});
```
When I select 'clients' - it loads content, menu item marked as 'active', everything works as expected.
When I select something (for example 'branch') from dropdown:
  - 'branch' content loads
  - 'branch' marked as active
  - 'clients' left marked as active
  - dropdown item not marked at all
  - URI changed to '/admin/#/branch'

Same thing if I then click to 'orders' - it marked as active, 'clients' unmarked, but old item from dropdown left active.

I understand that magic on this planet is limited, so I do not expect dropdown itself to be marked and unmarked as 'active', but items itself should change it status despite of its placement (inside dropdown or not), isn't it?

Maybe I'm doing something wrong?

**Comment by king2** [comment]
https://jsfiddle.net/omLgpq3n/17/

**Comment by Banandrew** [comment]
Hi @king2, sorry for the delay, and thank you for the test case. You can use `deactivate` property set to `'all'` during the tab initialization to fix #2, but the component doesn’t expect this use case, so you’ll have to somehow improvise—utilize the dropdown component’s callbacks, maybe its `action` property that can also accept a function, maybe use `$.tab('change tab', dataTab)` method to create your own logic.

If you have more questions or want me to elaborate, I’ll be happy to answer them either in this thread, or on Gitter.
